src/test/java/com/microsoftopentechnologies/windowsazurestorage/ConfigAsCodeTest.java

```diff
@@ -33,12 +33,13 @@ public void should_support_configuration_as_code() {
         assertThat(credentials.getDescription(), is("Account"));
         assertThat(credentials.getStorageAccountName(), is("a-storage-account"));
         assertThat(credentials.getBlobEndpointURL(), is("https://a-storage-account.blob.core.windows.net/"));
+        assertThat(credentials.getCdnEndpointURL(), is("https://cdn-resource-name.azureedge.net/"));
         assertThat(credentials.getId(), is("storage-account"));
         assertNotNull(credentials.getStorageKey());
     }
 
     @Test
-    @ConfiguredWithCode("configuration-as-code.yml")
+    @ConfiguredWithCode("com/microsoftopentechnologies/windowsazurestorage/configuration-as-code.yml")
```

Comment:
why?

Reply:
I couldn't get the ConfigAsCodeTest to run successfully in my local instance until I changed this. It was complaining about an incorrect path. Changing this fixed it for me locally.

Reply:
Here's the errors I am getting when running `mvn verify` with the default `configuration-as-code.yml` path:
```
[ERROR] Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 19.23 s <<< FAILURE! -- in com.microsoftopentechnologies.windowsazurestorage.ConfigAsCodeTest
[ERROR] com.microsoftopentechnologies.windowsazurestorage.ConfigAsCodeTest.should_support_configuration_as_code -- Time elapsed: 15.08 s <<< FAILURE!
java.lang.AssertionError: Exception when accessing the resources: configuration-as-code.yml
        at io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule.lambda$before$0(JenkinsConfiguredWithCodeRule.java:41)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule.before(JenkinsConfiguredWithCodeRule.java:44)
        at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:649)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.lang.Thread.run(Thread.java:829)

[ERROR] com.microsoftopentechnologies.windowsazurestorage.ConfigAsCodeTest.export_configuration -- Time elapsed: 4.124 s <<< FAILURE!
java.lang.AssertionError: Exception when accessing the resources: configuration-as-code.yml
        at io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule.lambda$before$0(JenkinsConfiguredWithCodeRule.java:41)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule.before(JenkinsConfiguredWithCodeRule.java:44)
        at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:649)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.lang.Thread.run(Thread.java:829)
```